src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp

```diff
@@ -903,34 +895,34 @@ Dictionary AdaptiveTileRendererFactory::get_params_metadata()
         "batch_size",
         Dictionary()
             .insert("type", "int")
-            .insert("default", "8")
+            .insert("default", "16")
             .insert("label", "Batch Size")
             .insert("help", "How many samples to render before each noise calculation"));
 
+    metadata.dictionaries().insert(
+        "min_samples",
+        Dictionary()
+            .insert("type", "int")
+            .insert("default", "0")
+            .insert("label", "Min Samples")
+            .insert("help", "Amount of uniform samples to render before adaptive samples"));
+
     metadata.dictionaries().insert(
         "max_samples",
         Dictionary()
             .insert("type", "int")
             .insert("default", "256")
             .insert("label", "Max Samples")
-            .insert("help", "Maximum number of anti-aliasing samples"));
+            .insert("help", "Maximum number of anti-aliasing samples (0 for unlimited)"));
 
     metadata.dictionaries().insert(
         "noise_threshold",
         Dictionary()
             .insert("type", "float")
-            .insert("default", "5.0")
+            .insert("default", "1.0")
             .insert("label", "Noise Threshold")
             .insert("help", "Rendering stop threshold"));
```

Comment:
Can you think of a better help message here?

Reply:
"Convergance rate. This is the maximum amount of noise allowed for areas of the image." ?

Reply:
Something like that yes, maybe:

    "Maximum amount of noise allowed in the image"

?

Reply:
Yes I like it